RobotArmAPP/Classes/Playback.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Shapes;
+
+namespace RobotArmAPP.Classes
+{
+    class Playback
+    {
+        public void PlaybackFunction(bool playing, bool okToSend,int currentFrame, int repeatTimes, int[] currentFrameArray, List<int[]> framesList, DispatcherTimer playbackTimer, Rectangle Blocker1, Rectangle Blocker2, Rectangle Blocker3, TextBox RepeatTimesBox, TextBox FrameSpeedBox, TextBox DelayBox, ListView FramesListView, Slider Eixo1Slider, Slider Eixo2Slider, Slider Eixo3Slider, Slider Eixo4Slider, Slider GarraSlider, Button StopPlayback)
+        {
+            FramesListView.SelectedIndex = currentFrame;
+            currentFrameArray = framesList[currentFrame];
+            GarraSlider.Value = currentFrameArray[0];
+            Eixo4Slider.Value = currentFrameArray[1];
+            Eixo3Slider.Value = currentFrameArray[2];
+            Eixo2Slider.Value = currentFrameArray[3];
+            Eixo1Slider.Value = currentFrameArray[4];
+            FrameSpeedBox.Text = Convert.ToString(currentFrameArray[5]);
+            DelayBox.Text = Convert.ToString(currentFrameArray[6]);
+            playbackTimer.Interval = TimeSpan.FromMilliseconds(currentFrameArray[6]);
+
+            if (framesList.Count > currentFrame + 1)
+            {
+                currentFrame++;
+            }
+            else
+            {
+                if (Convert.ToInt32(RepeatTimesBox.Text) == 0 && repeatTimes == 0)
+                {
+                    currentFrame = 0;
+                }
+                else if (Convert.ToInt32(RepeatTimesBox.Text) > 1)
+                {
+                    currentFrame = 0;
+                    RepeatTimesBox.Text = Convert.ToString(Convert.ToInt32(RepeatTimesBox.Text) - 1);
+                }
+                else
+                {
+                    RepeatTimesBox.Text = Convert.ToString(repeatTimes);
+                    IsPlaying(false,playing, okToSend, framesList,playbackTimer,Blocker1,Blocker2,Blocker3,StopPlayback);
+                }
+            }
+        }
+
+        public void IsPlaying(bool onOff,bool playing, bool okToSend, List<int[]> framesList, DispatcherTimer playbackTimer, Rectangle Blocker1, Rectangle Blocker2, Rectangle Blocker3, Button StopPlayback)
```

Comment:
Aqui uma questão de nomeação. Toda função que começa com `is` normalmente retorna um `bool`. Por exemplo, `isNumberGreaterThan100(int number)`.
```suggestion
        public void SetPlayingStatus(bool onOff,bool playing, bool okToSend, List<int[]> framesList, DispatcherTimer playbackTimer, Rectangle Blocker1, Rectangle Blocker2, Rectangle Blocker3, Button StopPlayback)
```

Reply:
Feito!